src/components/profile/ProfileDataTable.js

```diff
@@ -10,6 +9,7 @@ import InputRounded from '../common/form/InputRounded'
 import ErrorText from '../common/form/ErrorText'
 import Section from '../common/layout/Section'
 import { withStyles } from '../../lib/styles'
+import PhoneInput from './PhoneNumberInput/PhoneNumberInput'
```

Comment:
why was this moved here?
@MathiasMoralesDixtra

Reply:
Because that import generates crashes on native and time ago we decide to create a .web a .native file for the PhoneInputs, the result it's the same because it uses the same input for the web.

Reply:
i ment in the order of imports it seems like a redundant change.